### PR TITLE
Create Surigao City outline

### PR DESCRIPTION
**Pull request by FerminOcon**
This Pull Request closes issue #63 
### Adding some text
- Adding text

### CODE REVIEW

File: ferminocon-ph.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Surigao City :heart:
+
+#### Surigao City
```
Comment (FerminOcon) [inline ferminocon-ph.md:3]: test comment

Comment (cswartzentruber) [inline ferminocon-ph.md:3]: What is your favorite type of food to eat in the Phillipines?

File: ferminocon-ph.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Surigao City :heart:
```
Comment (ghost) [inline ferminocon-ph.md:1]: What country Surigao is?

Comment (FerminOcon) [inline ferminocon-ph.md:1]: @moyomendoza  Its Philippines

File: ferminocon-ph.md (hunk 1 above)
Comment (FerminOcon) [inline ferminocon-ph.md:3]: PInaksiw :100:

**Comment by FerminOcon** [comment]
:+1:

**Pull request merged**